README.md

```diff
@@ -120,6 +135,14 @@ $ dapr uninstall --all
 
 You should always run a `dapr uninstall` before running another `dapr init`.	
 
+#### Uninstall Dapr from a specific install path 
+
+If previously installed to a specific location, Dapr can be unsintalled with the `--install-path` argument
+
+```bash
+dapr uninstall --install-path ~/.dapr
```

Comment:
for this example, use some path other than ~/.dapr. As ~/.dapr is default folder for components on MAc.